packages/server/src/api/controllers/row/index.ts

```diff
@@ -146,6 +152,45 @@ export async function search(ctx: any) {
   })
 }
 
+function getSortOptions(
+  ctx: Ctx,
+  view: ViewV2
+):
+  | {
+      sort: string
+      sortOrder?: SortOrder
+      sortType?: SortType
+    }
+  | undefined {
+  const { sort_column, sort_order, sort_type } = ctx.query
+  if (Array.isArray(sort_column)) {
+    ctx.throw(404, "sort_column cannot be an array")
```

Comment:
Should these be a 404? Given that its an incorrect type - the API was found, but invalid inputs.

Reply:
Right, that was meant to be a 400